pinot-query-runtime/src/test/resources/queries/JoinQuery.json

```diff
@@ -0,0 +1,206 @@
+{
+  "basic_join_queries": {
```

Comment:
Tentatively change this as "FromExpressions.json` add additional tests in section 7.2.1